sample_apps/run_example.sh

```diff
@@ -10,19 +10,31 @@ Me=$(basename "$0")
 
 SampleAppName="simple_app"
 pushd "$(dirname "$0")" > /dev/null 2>&1
+
+# shellcheck disable=SC2046
+CERT_PROTO="$(dirname $(pwd))"; export CERT_PROTO
+
 Cleanup="$(pwd)/cleanup.sh"
 popd > /dev/null 2>&1
 
 MeMsg="Build and run example program"
 
 # Script global symbols to app-data dirs
-Client_app_data="./app1_data/"
-Srvr_app_data="./app2_data/"
+Client_app_data="app1_data"
+Srvr_app_data="app2_data"
```

Comment:
Sanitized: Now we have only one instance of these `app1_data` and `app2_data` in the scripts. 

Everywhere else, we now use the symbols.